### ISSUE TITLE
MessageCenterUpdatedEvent sends different values for ios and android

### ISSUE THREAD

**Issue by dkulakov** [urbanairship/react-native-airship#513]
❗For how-to inquiries involving Airship functionality or use cases, please
contact (support)[https://support.airship.com/].

# Preliminary Info

### What Airship dependencies are you using?

`"@ua/react-native-airship": "15.2.5"`

### What are the versions of any relevant development tools you are using?

# Report

### What unexpected behavior are you seeing?

`MessageCenterUpdatedEvent` sends different values for ios and android

for the android version the same properties are sent as specified in the interface `MessageCenterUpdatedEvent`
for the ios version I receive next properties:
```
{ messageCount: number, unreadCount: number }
```

### What is the expected behavior?

`MessageCenterUpdatedEvent` should send to ios and android the same properties as it mentioned at his interface
```
/**
 * Event fired when the Message Center  is updated.
 */
export interface MessageCenterUpdatedEvent {
  /**
   * The unread message count.
   */
  messageUnreadCount: number;
  /**
   * The total message count.
   */
  messageCount: number;
}
```

**Comment by rlepinski** [comment]
Thanks for the report, ill get it fixed in the next update

**Comment by rlepinski** [comment]
Fixed in 15.3.0
